examples/Atlas/urdf/atlas_minimal_contact.urdf

```diff
@@ -1,109 +1,87 @@
 <?xml version="1.0" ?>
 <!-- =================================================================================== -->
-<!-- |    This document was autogenerated by xacro from atlas.urdf.xacro               | -->
+<!-- |    This document was autogenerated by xacro from atlas_v5_minimal_contact.xacro | -->
 <!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
 <!-- =================================================================================== -->
-<robot name="atlas" xmlns:xacro="http://www.ros.org/wiki/xacro">
+<robot name="atlas" xmlns:xacro="http://ros.org/wiki/xacro">
   <link name="l_clav">
     <inertial>
-      <mass value="3.45"/>
-      <origin rpy="0 -0 0" xyz="0 0.048 0.084"/>
-      <inertia ixx="0.011" ixy="0" ixz="0" iyy="0.009" iyz="-0.004" izz="0.004"/>
-    </inertial>
-    <visual>
-      <origin rpy="0 -0 0" xyz="0 0 0"/>
-      <geometry>
-        <mesh filename="package://Atlas/urdf/meshes/l_clav.wrl" scale="1 1 1"/>
-      </geometry>
-    </visual>
-  </link>
-  <link name="l_farm">
-    <inertial>
-      <mass value="3.388"/>
-      <origin rpy="0 -0 0" xyz="0 -0.065 0"/>
-      <inertia ixx="0.00656" ixy="0" ixz="0" iyy="0.00358" iyz="0" izz="0.00656"/>
+      <mass value="4.466"/>
+      <origin rpy="0 0 0" xyz="0 0 -0.084"/>
+      <inertia ixx="0.011" ixy="0" ixz="0" iyy="0.009" iyz="0.004" izz="0.004"/>
     </inertial>
     <visual>
-      <origin rpy="0 -0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://Atlas/urdf/meshes/l_farm.wrl" scale="1 1 1"/>
+        <mesh filename="package://Atlas/urdf/meshes/r_clav.obj"/>
```

Comment:
As @tkoolen pointed out, the right-side meshes are the correct ones. The corresponding left-side meshes are mirrored components that do not exist. We have them in the drc repo because our model generation scripts copy all the meshes used in the OSRF urdfs, but they are not necessary.